### DISCUSSION TITLE
Practica HTML

### DISCUSSION THREAD

**Discussion by JulioRamnses** [Launch-X-Latam/MisionFrontEnd#313] [Q&A]
Gente vengo con una pregunta

¿la practica de la pastelería es a fuerza de una pastelería?

pregunto, por que, e trabajado en en proyecto de la uni que es de una pagina web que me a consumido el tiempo, entre cambios( que no eran muy grandes, pero si que afectaban el tiempo planeado) y ya no me da tiempo de hacer la pagina de la pastelería, ya que la pagina en la que e trabajado, es un proyecto del todo el salón, y bueno los trabajos escolares, no es que sean precisamente rápidos

entonces esa es mi duda, si la practica de HTML, es  si a fuerza es la pág. de la pastelería o puedo subir en la que estado trabajando junto con mis compañeros de clase

que tengan buen día, tarde o noches y disculpen las molestias

**Comment by RIvanCF** [comment]
Está dentro de los requisitos para que cumplas con la tarea. 
Te recomiendo leer el siguiente link 
Tienes hasta mañana para entregar la practica de HTML
https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML/practicas
